frontend/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:latest
```

Comment:
I took the same tag as the current circle-ci setup is using. @FredrikAugust Do you have any suggestions here?

Reply:
I would argue that always using the latest node version is better. It will most likely net us some performance improvements further down the line, and it should never break anything.